src/dms-preview/azext_dms/commands.py

```diff
@@ -25,5 +25,8 @@ def load_command_table(self, _):
         g.custom_command('create', 'create_or_update_project')
 
     with self.command_group('dms project task', dms_tasks_sdk, client_factory=dms_cf_tasks) as g:
+        g.custom_command('cancel', 'stop_task')
```

Comment:
Seems like `cancel` is unnecesary

Reply:
See my response to the other comment about the "cancel" command.